CMakeLists.txt

```diff
@@ -171,7 +171,7 @@ IF(CPUINFO_SUPPORTED_PLATFORM)
       LIST(APPEND CPUINFO_SRCS
         src/arm/android/properties.c)
     ENDIF()
-  ELSEIF(CMAKE_SYSTEM_NAME STREQUAL "Windows" AND CPUINFO_TARGET_PROCESSOR STREQUAL "ARM64")
+  ELSEIF((CMAKE_SYSTEM_NAME MATCHES "Windows") AND CPUINFO_TARGET_PROCESSOR STREQUAL "ARM64")
```

Comment:
Should be "^Windows"

Reply:
Nice. Fixed in e07e98b923d0e566a5a468d1cf3fd1c06c010757